### PR TITLE
7.0.0.rc0

### PR DESCRIPTION
**Pull request by ntindall**
I would like to release this as a candidate release version [note, not sure what is the most idiomatic ruby way to do this, please advise].

Please let me know if I have made any errors in these changes... I tried to keep the commits fairly isolated but admittedly I have never programmed in ruby / interacted with VCR, etc. before this change.

### CODE REVIEW

File: lib/plaid/version.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
 module Plaid
-  VERSION = '6.3.0'.freeze
-  API_VERSION = '2018-05-22'.freeze
+  VERSION = '7.0.0-rc0'.freeze
```
Comment (be9) [inline lib/plaid/version.rb]: `7.0.0.rc0`

File: test/vcr_cassettes/PlaidIncomeTest_test_get.yml [hunk 2]
```diff
@@ -140,4 +140,163 @@ http_interactions:
...
+      plaid-version:
+      - '2019-02-14'
+      vary:
+      - Accept-Encoding
+    body:
+      encoding: ASCII-8BIT
+      string: |-
+        {
+          "display_message": null,
+          "error_code": "PRODUCT_NOT_READY",
```
Comment (be9) [inline test/vcr_cassettes/PlaidIncomeTest_test_get.yml]: BTW what does that mean?

**Comment by be9** [comment]
Ah, one more thing @ntindall. Could you `rm -rf test/vcr_cassettes && STUB_API=1 RECORD_MODE=all rake test && git add --update test/vcr_cassettes`? VCR somehow started to append requests instead of overwriting, this is not what we want here. I'll look to update the docs later.

File: lib/plaid/version.rb (hunk 1 above)
Comment (be9) [reply lib/plaid/version.rb]: :zap:

File: test/vcr_cassettes/PlaidIncomeTest_test_get.yml (hunk 2 above)
Comment (ntindall) [reply test/vcr_cassettes/PlaidIncomeTest_test_get.yml]: it means that income isn't ready yet. we fire a webhook after it is ready. so the test would have to poll the endpoint if we wanted to test the income object.

File: lib/plaid/models.rb [hunk 3]
```diff
@@ -233,6 +233,44 @@ class NumberEFT < BaseModel
       property :institution
     end
 
+    # Public: A representation of an international account number.
```
Comment (skansal-plaid) [inline lib/plaid/models.rb]: We can add examples for `NumberInternational` and `NumberBACS` fields similar to examples for `NumberACH`. Can copy these examples from draft documentation for customers.

File: lib/plaid/models.rb [hunk 4]
```diff
@@ -359,13 +407,18 @@ class IdentityAddressData < BaseModel
...
       ##
       # :attr_reader:
-      # Public: The String ZIP code.
-      property :zip
+      # Public: The String postal code.
+      property :postal_code
+
+      ##
+      # :attr_reader:
+      # Public: The String country.
```
Comment (skansal-plaid) [inline lib/plaid/models.rb]: We can document country field as `ISO-3166-1 alpha-2`.

File: lib/plaid/models.rb [hunk 5]
```diff
@@ -674,8 +727,13 @@ class TransactionLocation < BaseModel
...
       ##
       # :attr_reader:
-      # Public: The String ZIP code (or nil).
-      property :zip
+      # Public: The String postal code (or nil).
+      property :postal_code
+
+      ##
+      # :attr_reader:
+      # Public: The String country.
```
Comment (skansal-plaid) [inline lib/plaid/models.rb]: Same comment - we can document country field as `ISO-3166-1 alpha-2`.

File: lib/plaid/models.rb [hunk 6]
```diff
@@ -674,8 +727,13 @@ class TransactionLocation < BaseModel
...
+      property :country
     end
```
Comment (skansal-plaid) [inline lib/plaid/models.rb]: Considering that the address fields (`city`, `region`, `street`, `postal_code`, and `country`) will now be nullable in version `2019-02-14`, are there changes we need to make here? cc @be9 there looks to be nullability logic around `Plaid.relaxed_models`?

Comment (ntindall) [reply lib/plaid/models.rb]: country is nullable in some of the address objects stored in the `vcr_cassettes` - so I am guessing there is nothing to do.

@be9 please confirm.

File: lib/plaid/models.rb (hunk 3 above)
Comment (ntindall) [reply lib/plaid/models.rb]: lets do it as a follow up (i'd want to use the same examples that we are going to publish on the docs).

**Comment by ntindall** [comment]
@be9 can you please publish this ? we may want to keep a `7.x` branch until we finalize the API verison.

File: lib/plaid/models.rb [hunk 7]
```diff
@@ -233,6 +233,8 @@ class NumberEFT < BaseModel
       property :institution
     end
 
+16830978
```
Comment (skansal-plaid) [inline lib/plaid/models.rb]: stray numbers?

File: lib/plaid/models.rb (hunk 6 above)
Comment (be9) [reply lib/plaid/models.rb]: It's OK. However we need tests for various country items.

**Comment by be9** [comment]
@ntindall :

> Pushed plaid 7.0.0.rc0 to https://rubygems.org

I haven't regenerated the docs, we better do this for the public release.

**Pull request merged**